CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# How to contribute
+
+This is an Open edX repo, and we welcome your contributions!
+Please read the [contributing guidelines](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/index.html).
+
+Here are some areas of heightened importance to consider when contributing to the UI Toolkit:
+
+- RTL
+  - Will the feature work for right-to-left languages?
+- a11y
+  - Is all functionality usable by using only the keyboard?
+  - Are there any special cases around focus management that need to be addressed?
```

Comment:
Similar to the keyboard bullet point, I'd also add a point here to make sure that the UI is accessible to screen reader users.

It might also be worth adding a sort of catch-all point that encourages the author to use aria attributes where applicable